qadence/backends/pyqtorch/backend.py

```diff
@@ -38,6 +39,51 @@
 logger = getLogger(__name__)
 
 
+def converted_circuit_with_noise(
```

Comment:
I am a little unclear on why we need this function. Is because of the logic on the functions in `qadence.execution`? E.g. the fact that they don't instantiate a `QuantumModel` but instead a backend directly.

Reply:
So it is because we can pass the noise in either the config, or as an argument of expectation or sample. For these last two, the input converted circuit may not have the noise set.